publish/dir.props

```diff
@@ -17,28 +17,23 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PublishRid Include="win7-x64"/>
-    <PublishRid Include="win7-x86"/>
-    <PublishRid Include="win8-arm"/>
-    <PublishRid Include="win10-arm64"/>
-    <PublishRid Include="osx.10.12-x64"/>
     <PublishRid Include="ubuntu.14.04-x64"/>
     <PublishRid Include="ubuntu.16.04-x64"/>
-    <PublishRid Include="ubuntu.14.04-arm"/>
-    <PublishRid Include="ubuntu.16.04-arm"/>
     <PublishRid Include="ubuntu.16.10-x64"/>
-    <PublishRid Include="rhel.7-x64"/>
     <PublishRid Include="debian.8-x64"/>
-    <PublishRid Include="fedora.23-x64"/>
-    <PublishRid Include="fedora.24-x64"/>
-    <PublishRid Include="opensuse.42.1-x64"/>
-    <PublishRid Include="linux-x64"/>
-    <PublishRid Include="linux-arm"/>
-    <PublishRid Include="linux-arm64"/>
-    <PublishRid Include="win-x86"/>
-    <PublishRid Include="win-x64"/>
+    <PublishRid Include="linux-x64_portable"/>
+    <PublishRid Include="win-x86_portable"/>
```

Comment:
Let's drop the "_portable".  The RID already tells us it is "portable" - "win" is portable; "osx" is portable; "linux" is portable.

I also think we should drop "portable" from the moniker as well.